example/build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'com.android.application'
+    id "com.diffplug.spotless" version "7.0.0.BETA2"
```

Comment:
💪 
Like that you're adding a linter to enforce code style consistency but typically those things are done best as separate PRs so linting changes are not intermixed with actual changes